Library/src/Multiplayer/SpaceEntitySystem.cpp

```diff
@@ -1368,7 +1383,6 @@ void SpaceEntitySystem::ApplyIncomingPatch(const signalr::value* EntityMessage)
 		const uint64_t EntityID = Deserialiser.ReadUInt64();
 		const uint64_t OwnerID	= Deserialiser.ReadUInt64();
 		const bool Destroy		= Deserialiser.ReadBool();
-		Deserialiser.Skip(); // ParentId //TODO: Support this if required
```

Comment:
How come we no longer need to at least skip the parent ID value?

Reply:
We handle this later on in DeserialiseFromPatch